docker-compose.yaml

```diff
@@ -26,6 +26,7 @@ services:
       - REDIS_PORT=6379
       - DELTALAKE_WAREHOUSE_DIR=s3a://cdm-lake/warehouse
       - SPARK_JOB_LOG_DIR=s3a://cdm-spark-job-logs/spark-job-logs
+      - SPARK_JOB_LOG_DIR_CATEGORY=master
```

Comment:
Why not just have a longer SPARK_JOB_LOG_DIR instead of adding another variable?

Reply:
It could give us more flexibilities - I am thinking in Jupyterhub, I can just set this to the username and each user will have their own log dir.

Reply:
Or I guess for workers automatically scaling you could dynamically set it to the hostname.

Reply:
yup. Although I think logs are only generated by the driver, we probably don't need to explicitly configure this for the workers/master.